.rubocop.yml

```diff
@@ -121,6 +121,8 @@ Style/PredicateName:
 Style/GuardClause:
   Exclude:
     - 'lib/active_fedora/autosave_association.rb'
+    - 'lib/active_fedora/file_configurator.rb'
```

Comment:
I'm not sure why that'd trigger a `Style/GuardClause`, but converting it to a `case` may be an easy improvement?